data/incipias/ships.txt

```diff
@@ -0,0 +1,50 @@
+ship "Nimbus"
+	"plural" "Nimbi"
+	"sprite" ship/nimbus
+	"thumbnail" thumbnail/nimbus
+	"swizzle" 0
+	attributes
```

Comment:
Uncapturable could make sense for the Nimbus, since the ship's life support is going to be providing a hydrogen atmosphere rather than an oxygen one.

Reply:
Yeah but when you buy it its been retrofitted to your needs
Although everyone has suits meaning they should be able to cap it anyway

Reply:
Yea, I am not sure what to assume here, has every crew member suits by default?
If not all their ships should probably be uncapturable.